### PR TITLE
Fix incorrect DOM nesting on Methodology page

### PR DESCRIPTION
**Pull request by Juneezee**
# Description and Motivation

Closes https://github.com/SatcherInstitute/health-equity-tracker/issues/2836.

## Has this been tested? How?

Check Console in Developer Tools to ensure no warnings are reported anymore

## Types of changes

- Bug fix

### CODE REVIEW

File: frontend/src/pages/Methodology/methodologySections/Covid19Link.tsx
Comment (Juneezee) [inline frontend/src/pages/Methodology/methodologySections/Covid19Link.tsx:1]: This fixes the following warning on http://localhost:3000/new-methodology/topics/covid#covid-19

```
Warning: validateDOMNesting(...): <ol> cannot appear as a descendant of <p>.
    at ol
    at p
    at article
    at section
    at Covid19Link
    at Route2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1777:29)
    at Switch2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1921:29)
    at article
    at div
    at div
    at section
    at div
    at MethodologyPage (http://localhost:3000/src/pages/Methodology/methodologyComponents/MethodologyPage.tsx:41:20)
```

File: frontend/src/pages/Methodology/methodologySections/PdohLink.tsx
Comment (Juneezee) [inline frontend/src/pages/Methodology/methodologySections/PdohLink.tsx:1]: This fixes the following warning on http://localhost:3000/new-methodology/topics/pdoh#pdoh

```
Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.
    at ul
    at p
    at article
    at section
    at PdohLink
    at Route2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1777:29)
    at Switch2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1921:29)
    at article
    at div
    at div
    at section
    at div
    at MethodologyPage (http://localhost:3000/src/pages/Methodology/methodologyComponents/MethodologyPage.tsx:41:20)
```

File: frontend/src/pages/Methodology/methodologySections/MetricsLink.tsx
Comment (Juneezee) [inline frontend/src/pages/Methodology/methodologySections/MetricsLink.tsx:1]: This fixes the first and second warnings on http://localhost:3000/new-methodology/definitions/metrics

```
Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    at div
    at DefinitionTooltip (http://localhost:3000/src/pages/Methodology/methodologyComponents/DefinitionTooltip.tsx:26:3)
    at p
    at div
    at article
    at section
    at MetricsLink
    at Route2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1777:29)
    at Switch2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1921:29)
    at article
    at div
    at div
    at section
    at div
    at MethodologyPage (http://localhost:3000/src/pages/Methodology/methodologyComponents/MethodologyPage.tsx:41:20)
```

```
Warning: validateDOMNesting(...): <table> cannot appear as a descendant of <p>.
    at table
    at http://localhost:3000/node_modules/.vite/deps/chunk-I77FE7FR.js?v=4261d6de:2703:50
    at Table2 (http://localhost:3000/node_modules/.vite/deps/chunk-4LDED4QY.js?v=4261d6de:103:17)
    at div
    at http://localhost:3000/node_modules/.vite/deps/chunk-I77FE7FR.js?v=4261d6de:2703:50
    at TableContainer2 (http://localhost:3000/node_modules/.vite/deps/chunk-WR5Z3EBI.js?v=4261d6de:80:17)
    at CodeBlock (http://localhost:3000/src/pages/Methodology/methodologyComponents/CodeBlock.tsx:25:3)
    at p
    at div
    at article
    at section
    at MetricsLink
    at Route2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1777:29)
    at Switch2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1921:29)
    at article
    at div
    at div
    at section
    at div
    at MethodologyPage (http://localhost:3000/src/pages/Methodology/methodologyComponents/MethodologyPage.tsx:41:20)
```

File: frontend/src/pages/Methodology/methodologyComponents/DefinitionTooltip.tsx
Comment (Juneezee) [inline frontend/src/pages/Methodology/methodologyComponents/DefinitionTooltip.tsx:1]: This fixes the third warning on http://localhost:3000/new-methodology/definitions/metrics

```
Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    at p
    at http://localhost:3000/node_modules/.vite/deps/chunk-I77FE7FR.js?v=4261d6de:2703:50
    at Typography2 (http://localhost:3000/node_modules/.vite/deps/chunk-P7KRRLUA.js?v=4261d6de:134:22)
    at span
    at div
    at DefinitionTooltip (http://localhost:3000/src/pages/Methodology/methodologyComponents/DefinitionTooltip.tsx:26:3)
    at p
    at div
    at article
    at section
    at MetricsLink
    at Route2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1777:29)
    at Switch2 (http://localhost:3000/node_modules/.vite/deps/chunk-I5VCZW3Z.js?v=4261d6de:1921:29)
    at article
    at div
    at div
    at section
    at div
    at MethodologyPage (http://localhost:3000/src/pages/Methodology/methodologyComponents/MethodologyPage.tsx:41:20)
```

**Review by Juneezee — COMMENTED** [review]
Tip: Hiding the whitespace changes can make the diff more readable

image

**Review by benhammondmusic — APPROVED** [review]
fantastic! thanks

**Pull request merged**